### PR TITLE
PR2 - Implemented checkers

### PR DESCRIPTION
**Pull request by iulianpascalau**
- implemented checkers
- added linter workflow

### CODE REVIEW

File: checkers/blsRatingsChecker.go [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+	}
+
+	allKeys := checker.hexBlsKeys
+	if len(extraBLSKeys) > 0 {
+		allKeys = append(allKeys, extraBLSKeys...)
+	}
+
+	log.Debug("blsRatingsChecker.Check", "checker name", checker.name, "num keys", len(allKeys))
+
+	response := make([]core.CheckResponse, 0)
```
Comment (andreibancioiu) [inline checkers/blsRatingsChecker.go:53]: Suggestion / optional renaming: `results := make([]core.CheckResult, 0)`.

**Review by andreibancioiu — DISMISSED** [review]

File: .github/workflows/golangci-lint.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+          # working-directory: somedir
+
+          # Optional: golangci-lint command line arguments.
+          args: --timeout 10m0s --max-issues-per-linter 0 --max-same-issues 0 --print-issued-lines
+
+          # Optional: show only new issues if it's a pull request. The default value is `false`.
+          only-new-issues: true
+
+          # Optional: if set to true then the action will use pre-installed Go
+          # skip-go-installation: true
```
Comment (miiu96) [inline .github/workflows/golangci-lint.yml:37]: add empty line

File: checkers/blsRatingsChecker.go [hunk 3]
```diff
@@ -0,0 +1,92 @@
+package checkers
+
+import (
+	"fmt"
+
+	"multiversx/mvx-keys-monitor/core"
```
Comment (miiu96) [inline checkers/blsRatingsChecker.go:6]: why this import doesn't contains `github.com` ?

File: checkers/blsRatingsChecker.go (hunk 1 above)
Comment (iulianpascalau) [reply checkers/blsRatingsChecker.go:53]: renamed to `results` :thum

File: .github/workflows/golangci-lint.yml (hunk 2 above)
Comment (iulianpascalau) [reply .github/workflows/golangci-lint.yml:37]: added

**Review by andreibancioiu — APPROVED** [review]

File: checkers/blsRatingsChecker.go (hunk 3 above)
Comment (iulianpascalau) [reply checkers/blsRatingsChecker.go]: renamed. Good catch :+1:

**Review by andreibancioiu — APPROVED** [review]

**Review by miiu96 — APPROVED** [review]

**Pull request merged**